src/components/ReportActionsSkeletonView/index.js

```diff
@@ -14,12 +14,12 @@ const propTypes = {
 
 const defaultProps = {
     shouldAnimate: true,
-    possibleVisibleContentItems: 0,
+    possibleVisibleContentItems: Math.ceil(Dimensions.get('window').height / CONST.CHAT_SKELETON_VIEW.AVERAGE_ROW_HEIGHT),
 };
 
 function ReportActionsSkeletonView({shouldAnimate, possibleVisibleContentItems}) {
     // Determines the number of content items based on container height
-    const visibleContentItems = possibleVisibleContentItems || Math.ceil(Dimensions.get('window').height / CONST.CHAT_SKELETON_VIEW.AVERAGE_ROW_HEIGHT);
+    const visibleContentItems = possibleVisibleContentItems;
```

Comment:
We don't need this new variable. Just use possibleVisibleContentItems

Reply:
👍🏻